msal/src/Microsoft.Identity.Client/Platforms/Android/UIParent.cs

```diff
@@ -25,68 +25,53 @@
 //
 //------------------------------------------------------------------------------
 
-#if ANDROID
+
```

Comment:
I've split UIParent.cs into platform specific UIParent because the class was too fragmented already.